src/components/ReturnListItem.vue

```diff
@@ -4,26 +4,37 @@
       <h2>{{ returnShipment.shipmentId }}</h2>
       <p>{{ returnShipment.shipmentItemCount }} {{ (returnShipment.shipmentItemCount > 1 ? 'Items' : 'Item') }}</p>
     </ion-label>
-    <ion-note slot="end">{{ returnShipment.estimatedArrivalDate ? ($filters.formatDate(returnShipment.estimatedArrivalDate)) : returnShipment.statusDesc }}</ion-note>
+    <ion-badge :color="statusColors[returnShipment.statusDesc]" slot="end">{{ returnShipment.estimatedArrivalDate ? ($filters.formatDate(returnShipment.estimatedArrivalDate)) : returnShipment.statusDesc }}</ion-badge>
```

Comment:
Remove estimated arrival date from the returns page and also handle the case when any specific status is not mapped with any color.

Reply:
Done sir